python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -17,7 +17,7 @@ class AutoMaterializePolicy(NamedTuple):
 
     - one or more of its parents is missing
     - the partition is a time window partition with a start time that is more than
-      time_window_partition_scope older than the latest available time window partition. By
+      time_window_partition_scope_minutes older than the latest available time window partition. By
```

Comment:
do you have a sense whether `time_window_partition_scope_minutes: float` (what we store in the NT) or `time_window_partition_scope: datetime.timedelta` (the arg to lazy and eager) is better? It seems unnecessary to have them be different

Reply:
I think I like minutes to match freshness